tribits/doc/sphinx/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==4.0.2
```

Comment:
This is a bit problematic.  I just installed Sphinx on my RHEL 7 machine at SNL with:

```
$ sudo yum install python-sphinx
```

and I got version 1.1.3:

```
$ sphinx-quickstart --version
Welcome to the Sphinx 1.1.3 quickstart utility.

Please enter values for the following settings (just press Enter to
accept a default value, if one is given in brackets).

Selected root path: --version

You have two options for placing the build directory for Sphinx output.
Either, you use a directory "_build" within the root path, or you separate
"source" and "build" directories within the root path.
```

:-(

I need to figure out how to install an updated version of Sphinx locally that can work with this.

Reply:
If you have pip with Python3, you can install Sphinx 4.1.0, even on RHEL 7.

Reply:
> If you have pip with Python3, you can install Sphinx 4.1.0, even on RHEL 7.

The time I created the script 4.0.2 was the latest version, it is a bad practice not to specify the exact version number as this may break compatibility as well as brings some security issues if pip is set to look somwhere else that pypi for packages.